TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleOpMain.java

```diff
@@ -0,0 +1,61 @@
+package org.firstinspires.ftc.teamcode;
+
+
+import com.arcrobotics.ftclib.command.CommandOpMode;
+
+import com.arcrobotics.ftclib.command.button.Button;
+import com.arcrobotics.ftclib.command.button.GamepadButton;
+import com.arcrobotics.ftclib.gamepad.GamepadEx;
+import com.arcrobotics.ftclib.gamepad.GamepadKeys;
+import com.arcrobotics.ftclib.hardware.motors.Motor;
+
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import org.firstinspires.ftc.teamcode.subsystems.DriveSystem;
+import org.firstinspires.ftc.teamcode.subsystems.Shooter;
+import org.firstinspires.ftc.teamcode.subsystems.commands.Com_Drive;
+import org.firstinspires.ftc.teamcode.subsystems.commands.Com_NoShoot;
+import org.firstinspires.ftc.teamcode.subsystems.commands.Com_Shoot;
+
+@TeleOp(name = "CommandBaseTest")
+public class TeleOpMain extends CommandOpMode {
+
+    private Motor fL, bL, fR, bR;
```

Comment:
Add a shooter motor to pass into the new constructor, reference one of the other comments.